src/components/course-administration/add-student.cjsx

```diff
@@ -0,0 +1,80 @@
+React = require 'react'
+BS = require 'react-bootstrap'
+_ = require 'underscore'
+{RosterActions, RosterStore} = require '../../flux/roster'
+{TutorInput} = require '../tutor-input'
+BindStoreMixin = require '../bind-store-mixin'
+
+BLANK_STUDENT = first_name: '', last_name: '', email: '', password: ''
+
+Field = React.createClass
+  propTypes:
```

Comment:
O cool! I didn't know that! maybe we should always assign it to a variable before exporting and then get rid of the `displayName` everywhere